project/config/hseni/config/hseni_theme.settings.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  node_user_picture: 1
+  comment_user_picture: true
+  comment_user_verification: true
+  favicon: 1
+logo:
+  use_default: 1
+favicon:
+  use_default: 1
+social_links_twitter: 'https://x.com/Hsenigov'
```

Comment:
Should these not be in config split so they can be edited on the production site without config changes being required?

Reply:
Done thank you Orla